lib/SPIRV/SPIRVReader.cpp

```diff
@@ -3513,6 +3513,12 @@ void generateIntelFPGAAnnotation(
   }
   if (E->hasDecorate(DecorationForcePow2DepthINTEL, 0, &Result))
     Out << "{force_pow2_depth:" << Result << '}';
+  if (E->hasDecorate(DecorationStridesizeINTEL, 0, &Result))
+    Out << "{stride_size:" << Result << "}";
+  if (E->hasDecorate(DecorationWordsizeINTEL, 0, &Result))
+    Out << "{word_size:" << Result << "}";
+  if (E->hasDecorate(DecorationTrueDualPortINTEL))
+    Out << "{true_dual_port:1}";
```

Comment:
it should be `{true_dual_port}` this annotation has no value associated with it

Reply:
The value part of attribute is following convention from RegistryINTEL and SimpleDualPortINTEL which also doesn't have value. Value of 1 in this case means enabled.

Reply:
@artemrad I agree with Bertrand, that the format that was set quite some time ago, you can see  [old translator PR](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/235) and [still existing test in clang](https://github.com/intel/llvm/blob/sycl/clang/test/CodeGenSYCL/intel-fpga-local.cpp#L21) for reference. Please reach us out if you would like to reconsider this format and refactor it.

Reply:
I disagree with the original request, since there is no scenario where `simple_dual_port:0` will get generated. This is redundant and confusing  translation of a SPIRV value-less decoration. The same logic applies to the new `true_dual_port`.

They should both have no value, to match with the SPIRV spec.

Reply:
@artemrad Done